src/frontend/staff-portal/src/environments/environment.prod.ts

```diff
@@ -1,4 +1,7 @@
 
 export const environment = {
-  production: true
+  production: true,
+  keycloakUrl: "https://dev.oidc.gov.bc.ca/auth", // change this for prod
```

Comment:
please talk to phil about the name of the keycloak server in the test environment and set these up so they point to test when the app is in test.  Alternatively, I think Phil is working on helm charts to write to the file keycloak-config.json, so settings can be pulled from there safely.